samples/03-configuration/README.md

```diff
@@ -90,19 +91,15 @@ Modify the code from the `HealthEndpointExtension.java` as shown below (use the
 of course):
 
 ```java
+@Requires(WebService.class)
```

Comment:
That's the old way to do it, no ?

Instead it should be:
```
@Inject
private WebService webService;
```

Reply:
Don't know if that can be called "old" but yes, `@Inject` is more used nowadays, replaced it also on sample-02.